sensorlib/src/CarlaCDASimAdapter.py

```diff
@@ -12,8 +12,9 @@
 from xmlrpc.server import SimpleXMLRPCServer
 import sys
 sys.path.append('../')
-from src.CarlaCDASimAPI import CarlaCDASimAPI
-from src.util.SimulatedSensorUtils import SimulatedSensorUtils
+sys.path.append('./')
+from CarlaCDASimAPI import CarlaCDASimAPI
```

Comment:
It looks like it is just selecting the current dicrectory. Do we actually need it?

Reply:
Let's remove this, it was fine without it.